test/utils/dropsToXrp.ts

```diff
@@ -87,38 +87,38 @@ describe("Drops To XRP", function () {
   it("throws with an amount with too many decimal places", function () {
     assert.throws(() => {
       dropsToXrp("1.2");
-    }, /has too many decimal places/);
+    }, /has too many decimal places/u);
```

Comment:
Why are these regex strings? (Or are they some other kind of string? - I'm not quite sure what /.../u does)

Reply:
It's regex - it checks that the regex is in the error message somewhere